modules/simphera_aws_instance/minio-storage.tf

```diff
@@ -1,7 +1,8 @@
 resource "aws_iam_role" "minio_iam_role" {
   name        = "${local.instancename}-s3-role"
   description = "IAM role for the MinIO service account"
-  tags        = var.tags
+  #depends_on  = [aws_iam_policy.minio_policy]
```

Comment:
remove line?

Reply:
We currently use this dependency on a branch for SIMPHERA. Leaving the comment in the code makes it easier to create a new SIMPHERA specific branch again.